scripts/release/build/build_info.py

```diff
@@ -57,7 +57,7 @@ def load_build_info(scenario: BuildScenario,
     f"""
     Load build information based on the specified scenario.
 
-    :param scenario: BuildScenario enum value indicating the build scenario (e.g., PATCH, STAGING, RELEASE).
+    :param scenario: BuildScenario enum value indicating the build scenario (e.g. "development", "patch", "staging", "release"). "development" scenario will return build info for "patch" scenario.
```

Comment:
nit: do we really need to introduce a new build scenario `development` if we are just treating it as `patch`. Should we consider adding a comment on the method that for development just pass patch?